hydrolib/core/dflowfm/extold/__init__.py

```diff
@@ -1,19 +1,17 @@
 from .models import (
```

Comment:
`ExtOldExtrapolationMethod` should also be exposed, right?

Reply:
Done!